query-node/schema.graphql

```diff
@@ -57,35 +57,29 @@ type ChannelCategory @entity {
 
 "Asset availability representation"
 enum AssetAvailability {
-  ACCEPTED, # asset is available in storage
-  PENDING, # asset is being uploaded to storage
-  URL, # asset is referencing URL
-  INVALID, # invalid storage (meta)data used
-}
+  "Asset is available in storage"
+  ACCEPTED,
 
-"Asset representation"
-union Asset = AssetUrl | AssetStorage | AssetDeleted
+  "Asset is being uploaded to storage"
+  PENDING,
 
-"Asset stored at an external source"
-type AssetUrl @variant {
-  "The http url pointing to the media"
-  url: String!
-}
+  "Asset is referencing URL"
+  URL,
 
-"Asset was deleted and is no longer available."
-type AssetDeleted @variant {
-  happenedIn: BigInt!
+  "Anvalid storage (meta)data used"
+  INVALID,
 }
 
-"Storage asset"
-type AssetStorage @variant {
-  """
-  Data object in upload life-cycle.
-  If this is deleted, then set oldDataObject in its place if it is set and not rejected, otherwise union goes to Deleted.
-  """
-  dataObject: AssetDataObject!
+"Asset representation"
+type Asset @entity {
```

Comment:
I don't think this is needed any longer.

Reply:
`Asset` has been removed.